Project Academy/10. Room-test/kotlin/app/src/test/java/com/dicoding/academies/data/AcademyRepositoryTest.kt

```diff
@@ -99,4 +100,20 @@ class AcademyRepositoryTest {
         assertNotNull(courseEntities.data?.mCourse?.title)
         assertEquals(courseResponses[0].title, courseEntities.data?.mCourse?.title)
     }
+
+    @Test
+    fun setCourseBookmark() {
+        val dummy = DataDummy.generateDummyCourses()[0]
+
+        academyRepository.setCourseBookmark(dummy, true)
+        verify(local, times(1)).setCourseBookmark(dummy, true)
+    }
+
+    @Test
+    fun setReadModule() {
+        val dummy = DataDummy.generateDummyModules("a")[0]
```

Comment:
Ini dummy cuman buat invoke sama verify? Tambahkan assertion

Reply:
Kalau yang ini kan yang di test method void dan hanya meneruskan call saja ke repository. Jadi menurutku, disini cukup verify call method saja
`https://blog.knoldus.com/unit-testing-void-methods-with-mockito-and-junit/` 

Atau mungkin dari mas Alfian ada cara lain kah ?